contentmgr/pinning.go

```diff
@@ -317,23 +317,26 @@ func (cm *ContentManager) UpdatePinStatus(location string, contID uint, status t
 			return errors.Wrap(err, "failed to look up content")
 		}
 
+		// if content is already active, ignore it
 		if c.Active {
-			return fmt.Errorf("got failed pin status message from location: %s where content(%d) was already active, refusing to do anything", location, contID)
+			return nil
```

Comment:
Did we remove this because of the logs or we just really want to ignore the active ones? Is there a way we can add this on the query filter before it gets here?

Reply:
This should not be happening, it means we sent a duplicate RPC message. If the pin is already active, no need to reactivate it. Better to ignore than complain, even though it is not actionable.